examples/interaction/js_callbacks/customjs_for_selection.py

```diff
@@ -1,3 +1,16 @@
+''' An interactive plot showcasing Bokeh's ability to add interactions
+using Custom Javascript.
+This example demonstrates selecting highlighted points on a graph.
```

Comment:
```suggestion
using Custom Javascript.

This example demonstrates selecting highlighted points on a graph.
```
maybe a blank line if this introduces a new paragraph